### PR TITLE
Define a placeholder for attachments

### LINKED ISSUE #243: Picture's display not smooth
Send or receive pictures through mms.

From the thread list, when opening a thread, the conversation appears abruptly.

 Also opening the picture can be improved, the header disappears too quickly.

### PR DESCRIPTION
**Pull request by lduboeuf**
This makes the conversation view rendering smoother

fixes: https://github.com/ubports/messaging-app/issues/243

Testing click artefact, see https://github.com/lduboeuf/messaging-app/actions

### CODE REVIEW

**Comment by UniversalSuperBox** [comment]
Hmm, I'm not sure how this actually helps the issue. Surely the page still freezes while the conversation loads because the Loaders aren't set `asynchronous: true`?

**Comment by lduboeuf** [comment]
Images are already loaded asynchronously, but as the wrapper around use the size of the image, the engine starts lately to draw the image and position it correctly, that makes the impression of ListView display texts first at the bottom and when comes the images, the conversation is moving up.

The idea is to define a default height to help the engine position early in the ListView, and that makes the conversation first display smoother. Scroll on conversation with images too.

File: src/qml/AttachmentsDelegate.qml [hunk 1]
```diff
@@ -112,13 +112,35 @@ Column {
...
+
     spacing: units.gu(1)
     Repeater {
         id: attachmentsRepeater
 
         Loader {
             id: attachmentLoader
             property bool loaded: status === Loader.Ready
+            height: loaded ? item.height : getMinHeight(modelData.type)
+            width: units.gu(27)
```
Comment (UniversalSuperBox) [inline src/qml/AttachmentsDelegate.qml:143]: Where does this width value come from?

Comment (lduboeuf) [reply src/qml/AttachmentsDelegate.qml:143]: All delegates have a max wifth set to 27 units. e.g: https://github.com/ubports/messaging-app/blob/xenial/src/qml/AttachmentDelegates/ImageDelegate.qml#L46

**Comment by Flohack74** [comment]
@UniversalSuperBox can we add this still for OTA-19 I think its a light change.

**Comment by lduboeuf** [comment]
i close that PR because i found a better way to do it

**Pull request closed without merging**